client/my-sites/navigation/index.jsx

```diff
@@ -19,17 +22,31 @@ class MySitesNavigation extends React.Component {
 	};
 
 	render() {
+		let sidebar = null;
```

Comment:
This kind of if/else doesn't really belong here. Also it's not clear to me yet that JPC and Calypso need a different sidebar object. Aren't we just re-using the same sidebar with different stuff enabled?

Reply:
See my comment https://github.com/Automattic/wp-calypso/pull/39413#discussion_r384432216